app/src/main/res/values-de/strings.xml

```diff
@@ -501,8 +501,8 @@
     <string name="label_system_message_team_member_left">%1$s wurde aus dem Team entfernt</string>
     <string name="label_system_message_user_not_available">Dieser Benutzer ist nicht mehr verfügbar</string>
     <string name="label_system_message_new_conversation_receipt_mode">Lesebestätigungen sind %1$s</string>
-    <string name="label_system_message_read_receipt_changed_by_self">%1$s haben Lesebestätigungen für alle %2$sgeschaltet</string>
-    <string name="label_system_message_read_receipt_changed_by_other">%1$s hat Lesebestätigungen für alle %2$sgeschaltet</string>
+    <string name="label_system_message_read_receipt_changed_by_self">%1$s turned read receipts %2$s for everyone</string>
+    <string name="label_system_message_read_receipt_changed_by_other">%1$s turned read receipts %2$s for everyone</string>
```

Comment:
English for DE is that correct ?

Reply:
wtf, I don't understand why this got mixed up with Crowdin 🤦 so no, it's broken. I will update it on their site and trigger a new one

Reply:
fixed